Client/Frontend/Browser/EnhancedTrackingProtection/SlideoverPresentationController.swift

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+struct SlideOverUXConstants {
+    static let ETPMenuHeightForGlobalOn: CGFloat = 385
+    static let ETPMenuHeightForGlobalOff: CGFloat = 285
+    static let ETPMenuCornerRadius: CGFloat = 8
+}
+
+class SlideOverPresentationController: UIPresentationController {
+
+    let blurEffectView: UIVisualEffectView!
+    var tapGestureRecognizer: UITapGestureRecognizer = UITapGestureRecognizer()
+    var globalETPStatus: Bool
+
+    init(presentedViewController: UIViewController, presenting presentingViewController: UIViewController?, withGlobalETPStatus status: Bool) {
+        self.globalETPStatus = status
+        let blurEffect = UIBlurEffect(style: .dark)
+        blurEffectView = UIVisualEffectView(effect: blurEffect)
+        super.init(presentedViewController: presentedViewController, presenting: presentingViewController)
+        tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(dismissController))
+        blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        self.blurEffectView.isUserInteractionEnabled = true
+        self.blurEffectView.addGestureRecognizer(tapGestureRecognizer)
```

Comment:
nit: Since parameter variables are named differently from class variables, we can probably remove `self` to keep it cleaner.